source/game_sa/Camera.cpp

```diff
@@ -826,7 +826,23 @@ void CCamera::SetColVarsAimWeapon(int32 aimingType) {
 
 // 0x50CC50
 void CCamera::SetColVarsPed(ePedType pedType, int32 nCamPedZoom) {
-    plugin::Call<0x50CC50, ePedType, int32>(pedType, nCamPedZoom);
+    if (pedType != PED_TYPE_PLAYER1 && pedType != PED_TYPE_PLAYER2) {
+        return;
+    }
```

Comment:
I'd prefer to use a `switch case here`
```suggestion
    switch (pedType) {
    case PED_TYPE_PLAYER1:
    case PED_TYPE_PLAYER2:
        break;
    default:
        return;
    }
```